package.json

```diff
@@ -22,14 +24,14 @@
   "author": "Dustin Deus <deusdustin@gmail.com> (http://www.starptech.de)",
   "license": "ISC",
   "dependencies": {
-    "chalk": "^2.3.0",
```

Comment:
Chalk 4 is Node10+. If this is a desired upgrade, adding *engines* metadata entry for visibility might be a good idea. And that would be semver major.